renderer/src/pages/Dashboard.tsx

```diff
@@ -35,29 +35,30 @@ const Dashboard = (): JSX.Element => {
     setTotalJobs(await getTotalJobsCompleted())
   }
 
-  const handleActivity = useCallback((value: ActivityEventMessage) => {
-    setActivities(activities => activities ? [...activities, value] : [value])
+  const handleActivity = useCallback((value: ActivityEventMessage[]) => {
+    setActivities(value)
   }, [])
 
   const handleEarnings = useCallback((value: number) => {
-    setTotalEarnigs((counter) => counter ? counter + value : value)
+    setTotalEarnigs(value)
   }, [])
 
   const handleJobs = useCallback((value: number) => {
-    setTotalJobs((counter) => counter ? counter + value : value)
+    setTotalJobs(value)
   }, [])
 
-  useEffect(() => { document.title = 'Dashboard' })
-
   useEffect(() => {
     reload()
-    startSaturnNode()
+    // startSaturnNode()
+    // return stopSaturnNode()
```

Comment:
@juliangruber I'm not sure if this should be done here in the dashboard, to ensure station starts and stops running on mount and on dismount. Somehow, it's related with`window.electron.saturnNode.start()` on `WalletConfig.tsx`. What do you think?

Reply:
If we change `WalletConfig.tsx` to only start Saturn when the address changes, and start Saturn initially when the `App` gets mounted (as long as there's an address configured), I think we won't have any extra start events. I'm going to PR it to this branch.

Reply:
I committed it right to this branch in https://github.com/filecoin-project/filecoin-station/pull/144/commits/1720b0fab52cc37dea44e93b8f6eb64ebdc750a0, as I can't file PRs against your fork unfortunately